akka-docs/src/main/paradox/scala/actors.md

```diff
@@ -169,9 +168,9 @@ which simultaneously safe-guards against these edge cases.
 
 #### Recommended Practices
 
-It is a good idea to provide factory methods on the companion object of each
-`Actor` which help keeping the creation of suitable `Props` as
-close to the actor definition as possible. This also avoids the pitfalls
+It is a good idea to provide @scala[factory methods on the companion object of each
+`Actor`] @java[static factory methods for each `Actor`] which help keeping the creation of 
+suitable `Props` as close to the actor definition as possible. This also avoids the pitfalls
 associated with using the @scala[`Props.apply(...)`] @java[ `Props.create(...)`] method which takes a by-name
```

Comment:
Pretty sure the java variation does *not* take a by-name, given that there is no such thing in Java  ;)